src/components/ErrorContextProvider/ErrorContextProvider.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+export const ErrorContext = React.createContext({
+  errorMessage: ErrorMessage.None,
+  setErrorMessage: () => {},
+  onNewError: () => {},
+} as ErrorContextProps);
```

Comment:
```suggestion
export const ErrorContext = React.createContext<ErrorContextProps>({
  errorMessage: ErrorMessage.None,
  setErrorMessage: () => {},
  onNewError: () => {},
});
```
Don't use `as`, `createContext` accepts generic